scripts/setup.py

```diff
@@ -235,63 +235,57 @@ def run_custom_scripts():
         raise e
 
 
-def local_mounts(mountlist):
-    """convert network_storage list of mounts to dict of mounts,
-    local_mount as key
-    """
-    return {str(Path(m.local_mount).resolve()): m for m in mountlist}
+def mounts_by_local(mounts):
+    """convert list of mounts to dict of mounts, local_mount as key"""
+    return {str(Path(m.local_mount).resolve()): m for m in mounts}
 
 
-@lru_cache(maxsize=None)
```

Comment:
What are your thoughts around removing the cache. From poking around I suspect it is only producing a single cache hit on startup, but then I have to wonder why it was added.

The rest of the change looks good.

Reply:
> I have to wonder why it was added

I wonder the same, my conclusion is that it's been added for no good reason.